src/common/lib/pio_helper.h

```
@@ -23,8 +23,8 @@
 
 #include "hardware/pio.h"
 
-PIO find_available_pio(const pio_program_t *program);
-void pio_restart(PIO pio, uint sm, uint offset);
+PIO   find_available_pio(const pio_program_t* program);
+void  pio_restart(PIO pio, uint sm, uint offset);
```

Clang (14.0.6)

[warning] parameter name 'sm' is too short, expected at least 3 characters

(readability-identifier-length)